exotica_core/init/dynamic_time_indexed_shooting_problem.in

```diff
@@ -7,5 +7,7 @@ Required double tau;  // Discretisation timestep of the trajectory
 
 Optional Eigen::VectorXd Q = Eigen::VectorXd();
 Optional Eigen::VectorXd R = Eigen::VectorXd();
+Optional Eigen::VectorXd GoalState = Eigen::VectorXd();
```

Comment:
Hmm, I understand the desire for this in XML - it is the same as X_star[-1]. Maybe we need to reconsider this down the line when we are adding nonlinear terms etc.

Reply:
I changed it to set the last column of X_star now. What else do you suggest here?

Reply:
Ok